docs/marks/geo.md

```diff
@@ -128,6 +128,8 @@ Plot.plot({
 ```
 :::
 
+The geo mark doesn’t have **x** and **y** channels; to derive those, for example to add [interactive tips](https://observablehq.com/@observablehq/plot-maps-tips), you can apply a [centroid transform](../transforms/centroid.md) on the geometries.
```

Comment:
It would be nice to include an example of centroid + text for geo labels here, and to link to the tip mark documentation rather than the example notebook, since the tip mark documentation includes that example, plus a lot more information about “interactive tips”.

Reply:
Agreed; I additionally copied the (same) interactive tip example to the centroid page.